webpack.config.js

```diff
@@ -1,38 +1,68 @@
-const HtmlWebPackPlugin = require("html-webpack-plugin");
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const webpack = require('webpack');
 
-module.exports = {
+module.exports = (env = {}) => {
+
+  const {mode = 'development'} = env;
+
+  const isProd = mode === 'production';
+  const isDev = mode === 'development';
```

Comment:
Move these strings to variables because you use them more than once.

Reply:
Extracted to constants